src/main/resources/templates/activities.html

```diff
@@ -10,5 +10,7 @@
 		<div th:replace="fragments/header :: header"></div>
 		<h1>Activities</h1>
 		<button onclick="location.href='/bored/random'">Test</button>
+		<div th:replace="fragments/footer :: footer"></div>
```

Comment:
This is just a suggestion since they had a footer fragment but never used it so I added this to the activities.html file, they can do this for all of their HTML files